src/app.js

```diff
@@ -95,3 +99,18 @@ setInterval(function() {
 panel.on('click', 'select', updatePokemon);
 panel.on('longClick', 'select', function() { options.debug = !options.debug; updatePokemon(); });
 
+Pebble.addEventListener('showConfiguration', function(e) {
+  Pebble.openURL(clay.generateUrl());
+});
+
+Pebble.addEventListener('webviewclosed', function(e) {
```

Comment:
The callback is just here at the bottom of app.js its just what method to call. Currently I call "draw" but we could add "configureUI" or something like that.